media/MPQEditor/index.js

```diff
@@ -33,19 +33,27 @@ function main() {
       case "modelNodesChanged":
         handleModelNodesChanged(message.names);
         break;
+      case "modelGraphIsShown":
+        handleModelGraphIsShown(message.shown);
+        break;
       default:
         break;
     }
   });
 
   vscode.postMessage({ type: "requestDisplayMPQ" });
   vscode.postMessage({ type: "requestModelNodes" });
+  vscode.postMessage({ type: "showModelNodes" });
```

Comment:
Q) this will post in serial at once and the response may not guarantee in this order. Is this OK?

Reply:
> Q) this will post in serial at once and the response may not guarantee in this order. Is this OK?

Yes. It is ok. They are independent.
`showModelNodes` just opens another panel for selection of layers to edit.